app/main/views/services.py

```diff
@@ -305,15 +308,15 @@ def delete_draft_service(framework_slug, lot_slug, service_id):
                                 delete_requested=True))
 
 
-@main.route('/frameworks/<framework_slug>/submissions/documents/<int:supplier_id>/<document_name>', methods=['GET'])
+@main.route('/frameworks/submissions/<framework_slug>/submissions/<int:supplier_id>/<document_name>', methods=['GET'])
```

Comment:
I like `assets` - I thought it seemed odd to hang it off `frameworks` really.  Not that anyone will ever see the URL, but still... will fix now.